base/exports.jl

```diff
@@ -23,6 +23,7 @@ export
     Docs,
     Markdown,
     Threads,
+    Iterators,
```

Comment:
if you do `using Iterators` without the package installed will it behave differently than with?